### PR TITLE
New version: KhronosGroup.VulkanSDK version 1.3.268.0

### PR DESCRIPTION
**Pull request by scribam**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/123613)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

**Comment by stephengillie** [comment]
Hi @scribam,

Is a dependency needed for the above error?

**Comment by scribam** [comment]
Hi @stephengillie,

AFAIK, yes, the dependency is needed.
You can have a look at my previous PR https://github.com/microsoft/winget-pkgs/pull/118619 which also had some issues with automatic validation

**Comment by stephengillie** [comment]
Do you know the name or location of the dependency with `vulkan-1.dll`? We should add it to the manifest.

File: manifests/k/KhronosGroup/VulkanSDK/1.3.268.0/KhronosGroup.VulkanSDK.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+PackageLocale: en-US
+Publisher: LunarG Inc.
+PublisherUrl: https://www.lunarg.com/
+PublisherSupportUrl: https://vulkan.lunarg.com/issue/home
+PrivacyUrl: https://vulkan.lunarg.com/sdk/home#content/popup/privacy/lg
+PackageName: Vulkan SDK
+PackageUrl: https://vulkan.lunarg.com/sdk/home
+License: MIT
+LicenseUrl: https://vulkan.lunarg.com/software/license/vulkan-1.3.268.0-windows-license-summary.txt
+Copyright: Copyright (c) 2015-2022 LunarG, Inc.
```
Comment (sandercox) [inline manifests/k/KhronosGroup/VulkanSDK/1.3.268.0/KhronosGroup.VulkanSDK.locale.en-US.yaml:15]: I think this should be updated to 2015-2023

File: manifests/k/KhronosGroup/VulkanSDK/1.3.268.0/KhronosGroup.VulkanSDK.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: --accept-licenses --default-answer --confirm-command install
+  SilentWithProgress: --accept-licenses --default-answer --confirm-command install
+UpgradeBehavior: install
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```
Comment (sandercox) [inline manifests/k/KhronosGroup/VulkanSDK/1.3.268.0/KhronosGroup.VulkanSDK.installer.yaml:19]: The error @stephengillie was seeing starting `vkcube.exe` was due to the fact that the Vulkan Runtime was not installed. This is normally taken care of by your graphics card drivers but you can also update this manually. I've created a new PR for adding VulkanRT too #124997 and we could add that as a dependency for the SDK maybe?

**Review by sandercox — COMMENTED** [review]
Wanted to do a similar PR but found this one.

I wasn't able to recreate the issue that the github workflow is seeing regarding the vcruntime. When I run this all using the sandbox testing. Must note that I was seeing different C++ runtimes installed as well so that might be an issue that was resolved already.

Also I noticed that the Sandbox test tool gave errors on the `VK_SDK_PATH` variables, made a PR for this #124995 too to prevent that.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Comment by stephengillie** [comment]
* https://github.com/microsoft/winget-pkgs/issues/107639

**Comment by sandercox** [comment]
@stephengillie I think you were still gonna look into this right, or is there any action that can be taken to get this package moving?

**Comment by stephengillie** [comment]
Hi @sandercox,

This package was depending on https://github.com/microsoft/winget-pkgs/pull/124997, which had stalled as the interactive installer installs different files than the unattended installer. I suspect installer switches might need to be adjusted.

**Pull request closed without merging**